server/run-tests.sh

```diff
@@ -2,7 +2,7 @@
 
```

Comment:
The PR's title/description don't match a lot of the content.

Reply:
Should I broke this into separate PRs with titles/descriptions specific to the various changes?

Reply:
You can have this PR as "fix the previous PR", but then have the commits separate and isolated doing their own thing, and change the title/description of the PR to say that this is a set of commits.